realm-annotations-processor/src/test/resources/io/realm/BooleansRealmProxy.java

```diff
@@ -26,13 +24,47 @@
 import org.json.JSONObject;
 import some.test.Booleans;
 
-public class BooleansRealmProxy extends Booleans implements RealmObjectProxy {
+public class BooleansRealmProxy extends Booleans
+        implements RealmObjectProxy {
```

Comment:
This code is a correct answer to annotation processor.
So I think it's good to keep as generated.